kubernetes_deploy/staging/deployment.yml

```diff
@@ -33,20 +33,25 @@ spec:
         - name: DB_USERNAME
           valueFrom:
             secretKeyRef:
-              name: db
-              key: username
+              name: rds
+              key: database_username
+        - name: DB_NAME
+          valueFrom:
+            secretKeyRef:
+              name: rds
+              key: database_name
         - name: DB_PASSWORD
           valueFrom:
             secretKeyRef:
-              name: db
-              key: password
+              name: rds
+              key: database_password
         - name: DB_HOST
           valueFrom:
             secretKeyRef:
-              name: db
-              key: host
+              name: rds
+              key: rds_instance_address
```

Comment:
I liked the previous naming scheme of `db.host`, `rds.rds_instance_address` confuses me a bit and repeats the “group name”.

What do you think about going back to `db.host`?

Reply:
Yep, `host` makes sense to me